lona/html/node.py

```diff
@@ -10,6 +10,23 @@
 from lona.protocol import NODE_TYPE
 
 
+def parse_style_string(style_string: str) -> Dict[str, str]:
```

Comment:
Hm... Now I'm not so sure about this design decision.

Ok we have consistency between `Node.id_list`, `Node.class_list` and `Node.style`.
But there is no consistency between those properties and `Node.__init__`.

So this works well:
```py
Div(id='foo bar', _class='foo bar', style='color: red')
```
But this doesn't work:
```py
Div().id_list = 'foo bar'
Div().class_list = 'foo bar'
Div().style = 'color: red'
```

@fscherf What do you think?

From the other side `id` from constructor becomes `id_list`. This stresses out that it should be a list. So maybe it's ok.

Reply:
@maratori 

> From the other side id from constructor becomes id_list. This stresses out that it should be a list. So maybe it's ok.

That was what i thought too. I experimented with the corresponding Javascript API and the Lona API should mimic ``element.idList``, ``element.classList`` and ``element.style``. I think this implementation is closest to the Javascript equivalent.